### ISSUE TITLE
ResourceList.Item shortcutActions not always vertically centered (2.0.0-beta.17)

### ISSUE THREAD

**Issue by mbaumbach** [Shopify/polaris#302]
# Issue summary

`shortcutActions` added to a `ResourceList.Item` appear to be aligned near the bottom of a `ResourceList.Item` that has **no media** associated with it.

## Expected behavior

All `ResourceList.Item` shortcut action buttons should be vertically centered.

## Actual behavior

If a `ResourceList.Item` has a media `Avatar` or `Thumbnail` associated with it, the buttons are centered vertically, however if there's no media the buttons are aligned to the bottom of the item.

**With Media**

![ResourceListWithMedia](https://media.giphy.com/media/ZgSygDXzH0w9ZsMRtC/giphy.gif)

**Without Media**

![ResourceListWithoutMedia](https://media.giphy.com/media/1VTJ7WxorPSdozAqpb/giphy.gif)

## Steps to reproduce the problem

1. Create a `ResourceList` with at least one item in it
1. Make sure the `ResourceList.Item` has no media node assigned.
1. Hover over the item and see the button aligned to the bottom.

The following code reproduces the problem (I used create-react-app to create a basic React app)

```
import React, { Component } from 'react';
import {
  ResourceList,
  AppProvider,
  Page,
  Layout,
  Card
} from '@shopify/polaris';

export default class App extends Component {
  render() {
    return (
      <AppProvider>
        <Page title="Rush">
          <Layout sectioned>
            <Card sectioned>
              <ResourceList 
                resourceName={{ singular: 'band member', plural: 'band members' }}
                items={[
                  {
                    id: 1,
                    name: 'Alex Lifeson'
                  },
                  {
                    id: 2,
                    name: 'Geddy Lee'
                  },
                  {
                    id: 3,
                    name: 'Neil Peart'
                  }
                ]}
                renderItem={item => (
                  <ResourceList.Item 
                    id={item.id} 
                    accessibilityLabel={item.name}
                    shortcutActions={[
                      { content: 'Alert', onAction: () => alert(item.name) }
                    ]}>
                    <h3>{item.name}</h3>
                  </ResourceList.Item>
                )} />
            </Card>
          </Layout>
        </Page>
      </AppProvider>
    );
  }
}
```

## Specifications

- Polaris version: 2.0.0-beta.17
- React version: 16.3.2
- Browser: Chrome 66.0.3359.117 (Official Build) (64-bit)
- Device: MacBook Pro (15-inch, 2016)
- Operating System: macOS High Sierra 10.13.4

**Comment by mbaumbach** [comment]
Verified this is fixed in 2.0.0-rc.1. Thanks for the quick fix!

**Comment by dfmcphee** [comment]
Excellent. Thanks for the issue @mbaumbach. 👍
